### PR TITLE
#547 #544 #588 #562 #560

### PR DESCRIPTION
**Pull request by szczepanbarszczowski**

### CODE REVIEW

File: src/routes/Home/Home.module.scss [hunk 1]
```diff
@@ -100,3 +96,13 @@
     height: 160px;
   }
 }
+
+.paragraphBold {
```
Comment (ArekRado) [inline src/routes/Home/Home.module.scss]: revert class and breakpoint :D these class is used only on mobile

```scss
@include media-breakpoint-down(sm) {
    .paragraphBold {
     ....
    }
}
```

File: src/routes/Home/Home.module.scss [hunk 2]
```diff
@@ -100,3 +82,30 @@
...
+
+.paragraph {
+  @include media-breakpoint-up(sm) {
+    @include font(paragraph-light, $color-gray-dark, $align: center, $font-family: $font-family-light);
+    line-height: 34px;
+    padding-top: 42px;
+  }
+
+  @include media-breakpoint-down(sm) {
+    @include font(h5, $color-gray-dark, $align: center, $font-family: $font-family-light);
```
Comment (ArekRado) [inline src/routes/Home/Home.module.scss]: h6-light - will be better

File: src/routes/Home/Home.module.scss [hunk 3]
```diff
@@ -100,3 +82,30 @@
...
+    font-size: 14px;
+    font-weight: 300;
+    padding-top: 22px;
+    padding-bottom: 67px;
+  }
+}
+
+@include media-breakpoint-down(sm) {
+  .paragraphBold {
+    @include font(h5, $color-gray-dark, $align: center, $font-family: $font-family-semibold);
```
Comment (ArekRado) [inline src/routes/Home/Home.module.scss]: h6- will be better

File: src/routes/Home/components/Carousel/Carousel.js [hunk 4]
```diff
@@ -56,18 +56,26 @@ export class Carousel extends PureComponent {
     }
   }
 
+  disableDrag = e => e.preventDefault()
+
+  // getHeight = element => {
```
Comment (ArekRado) [inline src/routes/Home/components/Carousel/Carousel.js]: remove commented block

File: src/routes/Home/Home.module.scss (hunk 2 above)
Comment (szczepanbarszczowski) [reply src/routes/Home/Home.module.scss]: +1

File: src/routes/Home/Home.module.scss (hunk 3 above)
Comment (szczepanbarszczowski) [reply src/routes/Home/Home.module.scss]: h6 has different font-size for sm breakpoint. I will stick with my solution and move color to mixin.

**Pull request merged**